frontend/packages/process-editor/src/utils/processEditorUtils/processEditorUtils.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns true if the version is 8 or higher, and false otherwise
+ *
+ * @param version the version to check
+ *
+ * @returns boolean
+ */
+export const getIfVersionIs8OrNewer = (version: string): boolean => {
+  const firstNumber: number = Number(version[0]);
+  return firstNumber > 7;
+};
```

Comment:
What about rename this to "supportsProcessEditor"? 🤔 Then we do not need to remember why we have this versionIs8orNewer method.

Reply:
Eks.

```suggestion
export const supportProcessEditor = (version: string): boolean => {
  const minimumSupportedVersion = 8;
  const currentVersion: number = Number(version[0]);
  return currentVersion >= minimumSupportedVersion ;
```